### PR TITLE
Load single best model as fallback

### PR DESCRIPTION
**Pull request by mfeurer**
in case user sets `ensemble_size=0` so that predict can always be used. Helpful for testing #1475.

### CODE REVIEW

**Review by eddiebergman — APPROVED** [review]

File: autosklearn/estimators.py [hunk 1]
```diff
@@ -526,8 +527,7 @@ def fit_ensemble(
         All parameters are ``None`` by default. If no other value is given,
         the default values which were set in a call to ``fit()`` are used.
 
-        Calling this function is only necessary if ``ensemble_size==0``, for
-        example when executing *auto-sklearn* in parallel.
+        Calling this function is only necessary if ``ensemble_size==0``.
```
Comment (KEggensperger) [inline autosklearn/estimators.py:530]: Is this true?

File: autosklearn/automl.py [hunk 2]
```diff
@@ -1554,65 +1554,55 @@ def _load_models(self):
...
-            self.ensemble_ = self._load_best_individual_model()
-
-        if self.ensemble_:
-            identifiers = self.ensemble_.get_selected_model_identifiers()
-            self.models_ = self._backend.load_models_by_identifiers(identifiers)
-
-            if self._resampling_strategy in ("cv", "cv-iterative-fit"):
-                self.cv_models_ = self._backend.load_cv_models_by_identifiers(
-                    identifiers
+        if (
```
Comment (KEggensperger) [inline autosklearn/automl.py:1557]: Can this be simplified and just checking whether ensemble_size == 0?

Comment (mfeurer) [reply autosklearn/automl.py:1557]: I am not sure, so I'd rather keep it.

Comment (mfeurer) [reply autosklearn/automl.py:1557]: Probably not, I just removed this statement.

**Pull request merged**